paddlenlp/layers/lora.py

```diff
@@ -401,3 +547,48 @@ def mark_only_lora_as_trainable(self) -> None:
                         weight.stop_gradient = False
                     else:
                         weight.stop_gradient = True
+
+    def get_lora_model(self, model: Union[PretrainedModel, nn.Layer], lora_config: LoRAConfig):
+
+        if lora_config.target_modules is None:
+            return model
+        elif isinstance(lora_config.target_modules, str):
+            target_modules = [lora_config.target_modules]
+            if lora_config.enable_lora_list is None or (
+                isinstance(lora_config.enable_lora_list, List)
+                and all(isinstance(item, bool) for item in lora_config.enable_lora_list)
+            ):
+                enable_lora_list = [lora_config.enable_lora_list]
+            else:
+                raise TypeError(
+                    f"Invalid lora_config.enable_lora_list value: {lora_config.enable_lora_list}. Since lora_config.target_modules is {str}, lora_config.enable_lora_list must be None or a list of {bool}"
+                )
+        else:
+            target_modules = lora_config.target_modules
+            if lora_config.enable_lora_list is None:
+                enable_lora_list = [None for _ in range(len(target_modules))]
+            elif isinstance(lora_config.enable_lora_list, List):
+                enable_lora_list = lora_config.enable_lora_list
+                if len(enable_lora_list) != len(target_modules):
+                    raise TypeError(
+                        f"Invalid lora_config.enable_lora_list value: {lora_config.enable_lora_list}. Since lora_config.target_modules is {List[str]}, len(enable_lora_list) should equal to len(target_modules): {len(enable_lora_list)} != {len(target_modules)}"
```

Comment:
```suggestion
                        f"Invalid lora_config.enable_lora_list value: {lora_config.enable_lora_list}. Since lora_config.target_modules is `List[str]`, `enable_lora_list` should have the same length as `target_modules`"
```